boards/matrix_portal_m4/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "matrix_portal_m4"
+version = "0.1.0"
+authors = ["Salsa Steve <elmanstevelaguna@gmail.com>"]
+description = "Board Support crate for the Matrix Portal M4"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
+categories = ["embedded", "hardware-support", "no-std"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+edition = "2021"
+
+# for cargo flash
+[package.metadata]
+chip = "ATSAMD51J19A"
+
+[dependencies.cortex-m-rt]
+version = "0.7"
+optional = true
+
+[dependencies.atsamd-hal]
+path = "../../hal"
```

Comment:
Specify a `crates.io` version for`atsamd-hal` rather than a path dependency

Reply:
Removed `path = "../../hal"`